code/modules/projectiles/tracer.dm

```diff
@@ -0,0 +1,38 @@
+/obj/effect/projectile
+	name = "pew"
+	icon = 'icons/obj/items/projectiles.dmi'
+	icon_state = ""
```

Comment:
why not make this an atom/movable instead

Reply:
this doesn't move

Reply:
the mor vars an obect has the longer it takes to New() which is why i ask